app/controllers/bookings_controller.rb

```diff
@@ -6,20 +6,37 @@ def index
   end
 
   def show
+    @booking = Booking.find(params[:id])
   end
 
   def new
+    @booking = Booking.new
   end
 
   def create
-  end
+    @booking = Booking.new
+
+    @lesson = Lesson.find(params[:lesson_id])
+    if @lesson.start_date <= current_user.user_plans.first.end_date && current_user.can_make_bookings? && @lesson.can_be_booked?
+      @booking.user_plan = current_user.user_plans.first
+      @booking.lesson = @lesson
+      @booking.user = current_user
 
-  def destroy
+      if @booking.save
+        redirect_to personal_dashboard_path(current_user.id)
```

Comment:
I believe we should not pass (current_user.id) in the personal_dashboard_path (see routes). Does this redirecting work?